overlord/servicestate/quota_control.go

```diff
@@ -85,7 +86,7 @@ func quotaGroupsAvailable(st *state.State) error {
 // CreateQuota attempts to create the specified quota group with the specified
 // snaps in it.
 // TODO: should this use something like QuotaGroupUpdate with fewer fields?
```

Comment:
I think we should look into this TODO now that we have two string arrays as arguments

Reply:
Nice suggestion! I'll do this!

Reply:
Opened PR https://github.com/snapcore/snapd/pull/12325 for this